### PR TITLE
TST: Use Actions

### PR DESCRIPTION
**Pull request by pllim**
As advertised.

See https://github.com/pllim/astropy-changelog/actions (tests passed)

### CODE REVIEW

File: .github/workflows/ci_workflows.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
```
Comment (astrofrog) [inline .github/workflows/ci_workflows.yml:20]: I wonder whether for all these small package with a single CI build we should just use 3.x then don't need to update in future?

**Review by astrofrog — APPROVED** [review]
Small comment below but looks good otherwise

File: .github/workflows/ci_workflows.yml (hunk 1 above)
Comment (pllim) [reply .github/workflows/ci_workflows.yml:20]: Did that for my stuff but then things broke (or very slow) when `3.x` became 3.9 but numpy/scipy didn't catch up yet. 🤷

Comment (pllim) [reply .github/workflows/ci_workflows.yml:20]: But I guess should be ok for this one...

**Pull request merged**